sigar_windows.go

```diff
@@ -194,10 +194,7 @@ func (self *ProcState) Get(pid int) error {
 		errs = append(errs, errors.Wrap(err, "getParentPid failed"))
 	}
 
-	self.Username, err = getProcCredName(pid)
-	if err != nil {
-		errs = append(errs, errors.Wrap(err, "getProcCredName failed"))
-	}
+	self.Username, _ = getProcCredName(pid)
```

Comment:
Could you add a comment ehre on why the error is not handled? I worry if someone touches this later on again it it will be just readded and the bug will be reintroduced.